scalastyle-config.xml

```diff
@@ -94,7 +94,7 @@
  </check>
  <check level="warning" class="org.scalastyle.scalariform.NumberOfMethodsInTypeChecker" enabled="true">
   <parameters>
-   <parameter name="maxMethods"><![CDATA[30]]></parameter>
+   <parameter name="maxMethods"><![CDATA[40]]></parameter>
```

Comment:
Probably want to revert this...`PersistentActor` has more than 30 methods (which is a lot) but it's distracting to edit in the IDE. I think the fact it's setting off this scalastyle warning is a red flag and we should circle back and clean it up a bit if possible.